util/pace/util/initialization/__init__.py

```diff
@@ -1,2 +1,6 @@
 from .allocator import QuantityFactory
+from .fortran_restart_to_pace_variable_names import (
+    extra_restart_properties,
+    fortran_restart_to_pace_dict,
```

Comment:
I don't see this file committed, but it belongs in `pace.driver`. Can you put this data there instead?